plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/IcebergTableExecuteHandle.java

```diff
@@ -73,4 +74,14 @@ public IcebergTableExecuteHandle withProcedureHandle(IcebergProcedureHandle proc
                 procedureHandle,
                 tableLocation);
     }
+
+    @Override
+    public String toString()
+    {
+        return toStringHelper(this)
```

Comment:
This will be printed in the explain plan, so we should format this in a nicer way. Don't include the class names in the output. Look at `HiveTableHandle` for an example.